src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -422,11 +423,14 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
    */
   checkForDuplicateInTable (business: any): boolean {
     const checkDuplication = this.getAmalgamatingBusinesses.find((b: any) =>
-      (business.type === AmlTypes.LEAR && b.identifier === business.businessInfo.identifier) ||
-      (business.type === AmlTypes.FOREIGN && b.corpNumber === business.corpNumber)
+      (b.type === AmlTypes.LEAR && b.identifier === business.businessInfo?.identifier) ||
```

Comment:
This here was wrong. We need to check b since that's the type of `AmalgamatingBusinessIF` which has the `type` field.

Reply:
Oops, my fault. Good find 👍